ivy/functional/ivy/experimental/statistical.py

```diff
@@ -253,6 +253,68 @@ def nanmean(
     )
 
 
+@handle_out_argument
+@handle_nestable
+@handle_backend_invalid
+@handle_exceptions
+@to_native_arrays_and_back
+@handle_device
+def nanmin(
+    a: ivy.Array,
```

Comment:
let's name `a` to `x` instead as that's the naming we've used for other statistical methods like `min`, `mean`, `std`, etc.

Reply:
@vedpatwardhan there are other functions in the file which are defined as a. So should I change them as well?

Reply:
Yeah we could change the rest too, probably in a different PR, but `x` certainly seems more consistent with our API than `a` 😄

Reply:
Okay sure noted.